morpho5/geometry/functional.h

```diff
@@ -87,11 +89,11 @@
 #define EQUIELEMENT_ARGS               "EquiElArgs"
 #define EQUIELEMENT_ARGS_MSG           "EquiElement allows 'grade' and 'weight' as optional arguments."
 
-#define FLORYHUGGINS_ARGS               "FlryHggnsArgs"
-#define FLORYHUGGINS_ARGS_MSG           "FloryHuggins requires a reference mesh and allows 'grade', 'a', 'b', 'c' and 'phi0' as optional arguments."
+#define HYDROGEL_ARGS                  "HydrglArgs"
+#define HYDROGEL_ARGS_MSG              "Hydrogel requires a reference mesh and allows 'grade', 'a', 'b', 'c' and 'phi0' as optional arguments."
 
-#define FLORYHUGGINS_PRP                "FlryHggnsPrp"
-#define FLORYHUGGINS_PRP_MSG            "FloryHuggins requires properties 'reference' to be a mesh, 'grade' to be an integer grade, 'a', 'b' and 'c' to be numbers and 'phi0' as either a number or a field."
+#define HYDROGEL_PRP                   "HydrglPrp"
+#define HYDROGEL_PRP_MSG               "Hydrogel requires properties 'reference' to be a mesh, 'grade' to be an integer grade, 'a', 'b' and 'c' to be numbers and 'phi0' as either a number or a field."
```

Comment:
I just realized that I forgot to update the error message according to the new functional constructor myself. Could you please change the part `'a', 'b' and 'c' to be numbers` to `'a', 'b', 'c', 'd' and 'phiref' to be numbers`?